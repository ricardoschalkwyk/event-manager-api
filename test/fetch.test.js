const supertest = require("supertest");

const app = require("../src/app");

describe("/api", () => {
  it("responds with a json message", async () => {
    const res = await supertest(app)
      .get("/api")
      .expect("Content-Type", /json/)
      .expect(200, {
        message: "API - 👋🌎🌍🌏",
      });

    expect(res.body).toMatchSnapshot();
  });
});
