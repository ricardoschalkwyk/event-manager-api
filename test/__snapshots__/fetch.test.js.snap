// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/api responds with a json message 1`] = `
Object {
  "message": "API - 👋🌎🌍🌏",
}
`;
